### ISSUE TITLE
PopUp Menu on top of the tables do not react

### ISSUE THREAD

**Issue by caunus** [ntop/ntopng#7448]
**Environment**:
* OS name: Ubuntu 
* OS version: 23.04 
* Architecture: 6.2.0-1004-raspi aarch64)
* ntopng version/revision: 5.2.220414 [Community build]

**What happened**:
Pop Up Menus on top of e.g. All Host table doesn't work - there is no reaction on mouse klick. only-text-buttons works fine but are rendered only as text

**How did you reproduce it?**
On every table with menu buttons

Happens on Ubuntu, MacOS, WIndows, iPadOS with Chrome, Edge, Safari (Mac and iPAdOS and Firefox Browser  

Looks like these menu button row:
<img width="902" alt="Bildschirmfoto 2023-05-04 um 18 51 41" src="https://user-images.githubusercontent.com/26174521/236274547-8ec791c4-ff9a-4f77-a757-e26d1235a851.png">

**Debug Information**:
I run ntopng with the fritzbox script. Status just direct after reboot. 

ntopng.service - ntopng - High-Speed Web-based Traffic Analysis and Flow Collection Tool
     Loaded: loaded (/lib/systemd/system/ntopng.service; enabled; preset: enabled)
     Active: active (running) since Thu 2023-05-04 18:50:04 CEST; 2min 54s ago
       Docs: man:ntopng(8)
             file:/usr/share/doc/ntopng/README.Debian
             file:/usr/share/doc/ntopng/UserGuide.pdf.gz
    Process: 1156 ExecStart=/usr/sbin/ntopng /etc/ntopng.conf (code=exited, status=0/SUCCESS)
   Main PID: 1407 (2/flow_checks)
      Tasks: 29 (limit: 3900)
        CPU: 1min 8.068s
     CGroup: /system.slice/ntopng.service
             └─1407 /usr/sbin/ntopng /etc/ntopng.conf

Mai 04 18:50:01 raspi4 ntopng[1156]: [Redis.cpp:99] ERROR: Connection error [Connection refused]
Mai 04 18:50:01 raspi4 ntopng[1156]: 04/May/2023 18:50:01 [Redis.cpp:99] ERROR: Connection error [Connection refused]
Mai 04 18:50:02 raspi4 ntopng[1156]: 04/May/2023 18:50:02 [Redis.cpp:83] Redis has disconnected, reconnecting [remaining attempts: 14]
Mai 04 18:50:02 raspi4 ntopng[1156]: 04/May/2023 18:50:02 [Redis.cpp:157] Successfully connected to redis 127.0.0.1:6379@0
Mai 04 18:50:02 raspi4 ntopng[1156]: 04/May/2023 18:50:02 [Redis.cpp:157] Successfully connected to redis 127.0.0.1:6379@0
Mai 04 18:50:02 raspi4 ntopng[1156]: [boot.lua:23] WARNING: No connectivity detected, ntopng will run in offline mode
Mai 04 18:50:02 raspi4 ntopng[1156]: 04/May/2023 18:50:02 [boot.lua:23] WARNING: No connectivity detected, ntopng will run in offline mode
Mai 04 18:50:03 raspi4 ntopng[1156]: 04/May/2023 18:50:03 [NetworkInterface.cpp:3226] Cleanup interface dummy
Mai 04 18:50:04 raspi4 ntopng[1156]: 04/May/2023 18:50:04 [Ntop.cpp:2365] Parent process is exiting (this is normal)
Mai 04 18:50:04 raspi4 systemd[1]: Started ntopng.service - ntopng - High-Speed Web-based Traffic Analysis and Flow Collection Tool.
Paste debug information below:

Browser Konsole show this error (on Chrome on Mac):

juery.js:3841 jQuery.Deferred exception: $(...).toast is not a function TypeError: $(...).toast is not a function
    at HTMLDivElement.<anonymous> (http://192.168.1.31:3001/js/ntop.min.js?1683219005:39:527)
    at Function.each (http://192.168.1.31:3001/js/jquery_bootstrap.min.js?1683219005:1:3898)
    at jQuery.fn.init.each (http://192.168.1.31:3001/js/jquery_bootstrap.min.js?1683219005:1:1923)
    at ToastUtils.initToasts (http://192.168.1.31:3001/js/ntop.min.js?1683219005:39:503)
    at HTMLDocument.<anonymous> (http://192.168.1.31:3001/lua/flows_stats.lua:831:15)
    at mightThrow (http://192.168.1.31:3001/js/jquery_bootstrap.min.js?1683219005:1:47303)
    at process (http://192.168.1.31:3001/js/jquery_bootstrap.min.js?1683219005:1:47957) undefined
jQuery.Deferred.exceptionHook @ jquery.js:3841
jquery.js:3850 Uncaught TypeError: $(...).toast is not a function
    at HTMLDivElement.<anonymous> (toast-utils.js:83:21)
    at Function.each (jquery.js:367:19)
    at jQuery.fn.init.each (jquery.js:202:17)
    at ToastUtils.initToasts (toast-utils.js:82:34)
    at HTMLDocument.<anonymous> (flows_stats.lua:831:15)
    at mightThrow (jquery.js:3557:29)
    at process (jquery.js:3625:12)

**Comment by MatteoBiscosi** [comment]
hi @caunus we do not support ntopng 5.2 version anymore. Please update to the latest version (5.6 or 5.7 version) and let me know if the problem persists

**Comment by caunus** [comment]
Hi @MatteoBiscosi thanks for your answer. This version 5.2 was installed with a fresh Ubuntu version 23.04 on my raspberry via apt install ntopng. Not sure how to update the version on my Raspi with Ubuntu because apt still only support 5.2. I tried the procedure for Raspberry and as well for Ubuntu on that page https://packages.ntop.org. It doesn't work out. So I tried the docker version for arm and there the menus at least work. Because I want to monitor my fritzbox and I'm not an expert on docker and have no clue how to integrate the script from your git repository. I think I have to wait until the Ubuntu repository will deliver a fresher version. Thanks again, Caunus
